react/components/DepartmentFilters.js

```diff
@@ -37,7 +37,7 @@ const DepartmentFilters = ({
       )}
       <div
         className={classNames(
-          styles.selectedCategoriesContainer,
```

Comment:
this is not a breaking change because this class was never declared on the .css file, so this line was useless before